packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -117,6 +117,7 @@ class CollapsibleMenuViewItem extends React.Component {
     } = this.props;
 
     const { isCollapsibleGroupItem, isCollapsibleMenuItem } = this.context;
+    const { isSelectable, isSelectedButton } = customProps;
```

Comment:
```suggestion
    const { isSelectable, isSelected } = customProps;
```
Let's keep the prop name consistent with terra-button

Reply:
We already have props with name "isSelected" in CollapsibleMenuViewItem ,so i require to create a prop with different name. By declaring the props with isSelected in customProps system is throwing the compilation error.